server/gc/src/main/java/org/apache/accumulo/gc/GCRun.java

```diff
@@ -492,6 +511,10 @@ static void minimizeDeletes(SortedMap<String,GcCandidate> confirmedDeletes,
         }
       }
     }
+    if (progressTimer.hasElapsed(Duration.ofMinutes(1))) {
```

Comment:
This is outside the while loop, was it intended to be inside?  If it is intended to be inside the while loop removes stuff from confirmedDeletes, so not sure if that matters for calling size() inside the loop.

Reply:
This was meant to be inside, moved it and also grabbed the map.size before the iterator loop so the total amount shouldn't change.